subworkflows/local/compute_trees.nf

```diff
@@ -20,17 +20,17 @@ include { CLUSTALO_GUIDETREE } from '../../modules/nf-core/clustalo/guidetree/ma
    ch_fastas_fortrees = ch_fastas.combine(tree_tools)
                                  .map( it -> [it[0] + it[2], it[1]] )
                                  .branch{
-                                          famsa_guidetree: it[0]["tree"] ==  "FAMSA_GUIDETREE"
-                                          mbed: it[0]["tree"] == "MBED"
+                                          famsa:    it[0]["tree"] == "FAMSA"
+                                          clustalo: it[0]["tree"] == "CLUSTALO"
                                         }
```

Comment:
```suggestion
  ch_fastas
      .combine(tree_tools)
      .map{ it -> [ it[0] + it[2], it[1] ] }
      .branch {
          famsa_guidetree: it[0]["tree"] ==  "FAMSA_GUIDETREE"
          mbed: it[0]["tree"] == "MBED"
          famsa:    it[0]["tree"] == "FAMSA"
          clustalo: it[0]["tree"] == "CLUSTALO"
      }
      .set { ch_fastas_fortrees }
```

Reply:
Yeah sorry, I copy paste it wrongly, I was complaining about the formatting of the code

Reply:
I updated the comment but double check I did it correctly

Reply:
It would not allow me to change it here directly. I changed it directly in the code and pushed again.